pkg/config/constants.go

```diff
@@ -0,0 +1,12 @@
+package config
+
+const (
+	OperatorNamespace = "OPERATOR_NAMESPACE"
+
+	ToolchainConfigMapName = "toolchain-saas-config"
+
+	ToolchainConfigMapUserApprovalPolicy = "user-approval-policy"
```

Comment:
This is documented in the detailed design - [https://docs.google.com/document/d/19UzaXVqiOo9JqDYz3qUKguJwW-7zceA4UtTr358R_CA/edit#heading=h.mciqf5191h4u](https://docs.google.com/document/d/19UzaXVqiOo9JqDYz3qUKguJwW-7zceA4UtTr358R_CA/edit#heading=h.mciqf5191h4u)

Reply:
ah, I forgot that it is in the detailed design doc. I thought that we have discussed that in the team, but it was probably only me & Alexey. The point is that it will be good to keep as much information as possible in the CRs that we can easily modify (PATCH) based on our needs (we can manage the system on our own without bothering SRE guys).

Reply:
This will require a change to the design then, let's address it separately to this.

Reply:
yup, I agree - that's why I mentioned that as "a side note" :-)